lib/astarte_vmq_plugin.ex

```diff
@@ -38,16 +40,7 @@ defmodule Astarte.VMQ.Plugin do
       # Not a device, let someone else decide
       :next
     else
-      subscriber_id = {mountpoint, username}
-      # TODO: we probably want some of these values to be configurable in some way
-      {:ok,
-       [
-         subscriber_id: subscriber_id,
-         max_inflight_messages: 100,
-         max_message_size: 65535,
-         retry_interval: 20000,
-         upgrade_qos: false
-       ]}
+      maybe_authorize_registration(mountpoint, username)
```

Comment:
You can drop the `maybe`, if a function is called `authorize` then it already means that it can either return `:ok` or some kind of auth error